projects/1.16/assets/fantasys-furniture/fantasyfurniture/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+{
+  "block.fantasyfurniture.decorations.berry_basket_blueberry": "蓝梅篮",
+  "block.fantasyfurniture.decorations.berry_basket_empty": "浆果篮",
+  "block.fantasyfurniture.decorations.berry_basket_strawberry": "草莓篮",
+  "block.fantasyfurniture.decorations.berry_basket_sweetberry": "甜莓篮",
+  "block.fantasyfurniture.decorations.bolts_of_cloth": "布匹",
+  "block.fantasyfurniture.decorations.book_stack": "书架",
+  "block.fantasyfurniture.decorations.book_stack.stacked": "堆积的书堆",
+  "block.fantasyfurniture.decorations.bowl_beetroot_soup": "",
+  "block.fantasyfurniture.decorations.bowl_empty": "碗",
+  "block.fantasyfurniture.decorations.bowl_mushroom_stew": "甜菜根汤碗",
+  "block.fantasyfurniture.decorations.coin_stack_gold": "金币堆",
+  "block.fantasyfurniture.decorations.coin_stack_iron": "铁币堆",
+  "block.fantasyfurniture.decorations.dunmer.pottery_0": "丹莫陶器 1",
+  "block.fantasyfurniture.decorations.dunmer.pottery_1": "丹莫陶器 2",
+  "block.fantasyfurniture.decorations.muffins_blueberry": "蓝莓松饼",
+  "block.fantasyfurniture.decorations.muffins_blueberry.stacked": "堆积的蓝莓",
+  "block.fantasyfurniture.decorations.muffins_chocolate": "巧克力松饼",
+  "block.fantasyfurniture.decorations.muffins_chocolate.stacked": "堆积的巧克力",
+  "block.fantasyfurniture.decorations.muffins_sweetberry": "甜莓松饼",
+  "block.fantasyfurniture.decorations.muffins_sweetberry.stacked": "堆积的甜莓",
+  "block.fantasyfurniture.decorations.mushrooms_red": "红蘑菇",
+  "block.fantasyfurniture.decorations.mushrooms_red.stacked": "堆积的红蘑菇",
+  "block.fantasyfurniture.decorations.nordic.boiled_creme_treats": "奶油煮零食",
+  "block.fantasyfurniture.decorations.nordic.boiled_creme_treats.stacked": "堆积的奶油煮零食",
+  "block.fantasyfurniture.decorations.nordic.mead_bottles": "蜂蜜酒瓶",
+  "block.fantasyfurniture.decorations.nordic.mead_bottles.stacked": "堆积的蜂蜜酒瓶",
+  "block.fantasyfurniture.decorations.nordic.soul_gems_dark": "黑色灵魂宝石",
+  "block.fantasyfurniture.decorations.nordic.soul_gems_light": "灵魂宝石",
+  "block.fantasyfurniture.decorations.nordic.sweetrolls": "甜卷",
+  "block.fantasyfurniture.decorations.nordic.sweetrolls.stacked": "堆积的甜卷",
+  "block.fantasyfurniture.decorations.paper_stack": "纸叠",
+  "block.fantasyfurniture.decorations.tankards_empty": "酒杯",
+  "block.fantasyfurniture.decorations.tankards_empty.stacked": "堆积的大啤酒",
+  "block.fantasyfurniture.decorations.tankards_honeymead": "蜂蜜酒",
+  "block.fantasyfurniture.decorations.tankards_honeymead.stacked": "堆积的蜂蜜酒",
+  "block.fantasyfurniture.decorations.tankards_milk": "牛奶啤酒",
```

```suggestion
  "block.fantasyfurniture.decorations.tankards_milk": "酒杯牛奶",
```
```suggestion
  "block.fantasyfurniture.decorations.tankards_milk": "牛奶酒杯",
```
tankard不一定是酒杯就是了，不过这样也行
同时所有语言文件中的Tankard最好译出